screens/ViewImageScreen.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from "react";
+import AsyncStorage from "@react-native-async-storage/async-storage";
+import { NavSubProps as RootNavSubProps } from "../App";
+import { Box, Center, HStack, Spinner, View, VStack } from "native-base";
+import { z } from "zod";
+import BlobMedia, { useMedia } from "../components/BlobMedia";
+import { Attachment } from "../lib/profile";
+import { Entry } from "../components/Form";
+import { Url } from "url";
+// @ts-ignore
+import { Image } from "native-base";
+
+/**
+ * The URL validator for the patient profile
+ */
+
+export default function ViewImageScreen({
+  route,
+  navigation,
+}: RootNavSubProps<"ViewImage">) {
+  const dataUri = useMedia(route.params.attachment.blob);
+
+  if (dataUri == null) {
+    return (
+      <Box>
+        <Spinner size="lg" />
+      </Box>
+    );
+  }
+
+  return (
+    <Center>
+      <Image
+        style={{ width: "100%", height: 500 }}
```

Comment:
Is there any way to make the image take up the entire screen vertically?

Reply:
I couldn't find anything in the docs. If you can feel free to change it though..!